curvefs/src/client/fuse_client.cpp

```diff
@@ -1471,5 +1548,32 @@ FuseClient::SetMountStatus(const struct MountOption *mountOption) {
     return CURVEFS_ERROR::OK;
 }
 
+void FuseClient::InitQosParam() {
```

Comment:
Add new code, please add the corresponding unit test.

Reply:
ok